iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/LineAdderImpl.java

```diff
@@ -36,6 +35,15 @@ class LineAdderImpl extends AbstractBranchAdder<LineAdderImpl> implements LineAd
     LineAdderImpl(NetworkImpl network, String subnetwork) {
         this.network = network;
         this.subnetwork = subnetwork;
+        this.copiedLine = null;
+    }
+
+    LineAdderImpl(NetworkImpl network, String subnetwork, Line copiedLine) {
+        this.network = network;
+        this.subnetwork = subnetwork;
+        this.copiedLine = copiedLine;
+        LineAdderImpl lineAdderImpl = this;
+        LineAdder.fillLineAdder(lineAdderImpl, copiedLine);
```

Comment:
This could be simplified:

```suggestion
        LineAdder.fillLineAdder(this, copiedLine);
```